assets/js/data/schema/README.md

```diff
@@ -35,9 +35,9 @@ wp.data.select( SCHEMA_STORE_KEY ).getRoute( '/wc/blocks', 'products', [20] );
 | Argument    | Type   | Description                                                    |
 | ----------- | ------ | -------------------------------------------------------------- |
 | `namespace` | string | Namespace for the route (eg. `/wc/blocks`)                     |
-| `modelName` | string | The model name for the route (eg. `products/attributes/terms`) |
+| `resourceName` | string | The resource name for the route (eg. `products/attributes/terms`) |
```

Comment:
Table got misaligned here too.

Reply:
I don't think it matters here because it's markdown and the rendered display is unaffected.